_includes/content/cla.md

```diff
@@ -1,7 +1,39 @@
-## Agreeing to the CLA
+## Contribution Methods
+
+To simplify the contribution process and encourage community contribution, we are gradually switching the contribution method from Contributor License Agreement (CLA) to Developer Certificate of Origin (DCO). See [our blog about switching to DCO](https://strongloop.com/strongblog/switching-to-dco/).
+
+### Developer Certificate of Origin (DCO)
+As an alternative to CLA, a Developer Ceritifcate of Origin (DCO) is a more lightweight contribution method. According to [Wikipedia](https://en.wikipedia.org/wiki/Developer_Certificate_of_Origin):
+
+> Instead a signed legal contract, a DCO is an affirmation that the source code being submitted originated from the developer, or that the developer has permission to submit the code.
+
+The full text of DCO can be found: https://developercertificate.org/.
+
+#### Signing off commits using DCO
+For GitHub repositories using DCO, you need to sign off each commit. To sign off, you can use the `-s` flag or adding `Signed-off-By: Name<Email>` in the commit message. For example, 
```

Comment:
How can contributors tell whether a repository is using DCO or CLA? Do they have to open a PR and then see the outcome of CLA/DCO check, or is there a better way?

Reply:
Currently the only way to tell before submitting a PR is to look up the `Contributing` section, but it's not ideal. Do you have other suggestion?

Reply:
Added the sentence to check the `Contributing` section.

Reply:
Sounds good for now 👍🏻